src/main/java/tracko/logic/commands/order/FindOrderCommand.java

```diff
@@ -16,21 +19,31 @@ public class FindOrderCommand extends Command {
 
     public static final String COMMAND_WORD = "findo";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all orders that contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " keychain";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Finds all orders that contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers."
+            + "Keywords under the same prefix are separated by a space. "
+            + "Use at least one prefix during searching. \n"
+            + "Parameters: \n"
+            + PREFIX_ITEM + "ITEM_KEYWORD [MORE_ITEM_KEYWORDS]...\n"
+            + PREFIX_ADDRESS + "ADDRESS_KEYWORD [MORE_ADDRESS_KEYWORDS]...\n"
+            + PREFIX_NAME + "NAME_KEYWORD [MORE_NAME_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD
+            + " " + PREFIX_ITEM + "keychain " + PREFIX_ADDRESS + "Ang Mo Kio Clementi";
 
-    private final OrderContainsKeywordsPredicate predicate;
+    public static final String INVALID_FLAG_MESSAGE = "Please use either -d/p or -D/P";
```

Comment:
You might want to consider using `-d / -p or -D / -P` instead, to deconflict between the appearance of a prefix (i.e, `n/`, `i/`, etc) and a flag as shown